stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -851,6 +851,9 @@ function(add_swift_target_library_single target name)
     list(APPEND SWIFTLIB_SINGLE_SWIFT_COMPILE_FLAGS "-Xfrontend" "-define-availability" "-Xfrontend" "${def}") 
   endforeach()
 
+  # Enable -target-min-inlining-version
+  list(APPEND SWIFTLIB_SINGLE_SWIFT_COMPILE_FLAGS "-Xfrontend" "-target-min-inlining-version" "-Xfrontend" "abi")
```

Comment:
What does `abi` mean in this context?

Reply:
`abi` is a placeholder for "the OS version at which Swift became ABI stable on this platform".

Reply:
Oh, that probably isn't right -- the 5.0 back deployment runtime allows Swift code to deploy back to a number of releases earlier than that!

(I think the earliest possible deployment target is `macOS 10.10, iOS 8.0, watchOS 2.0, tvOS 9.0`. The Swift 5 release was shipped in `macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2`.)

Reply:
(Perhaps we should have `min` alias instead that corresponds to those.)

Reply:
Ok, that makes sense. I'll experiment with adding that in this PR.